client/verta/verta/endpoint/resources/nvidia_gpu.py

```diff
@@ -67,9 +67,13 @@ def _to_hardware_compatibility_dict(self):
         if self.model is None:
             return {"nvidia_gpu": {"all": True}}
         else:
-            return {"nvidia_gpu": {self.model.value: True}}
+            return {"nvidia_gpu": {self.model.name: True}}
 
 
-class NvidiaGPUModel(str, Enum):
```

Comment:
Subclassing `str` was causing ReadTheDocs to document `str`'s methods.

Removing `str` means we have to use `.name` rather than `.value` above.